streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -796,3 +815,20 @@ package object testPackage extends Assertions {
     }
   }
 }
+
+/**
+ * Helper methods for testing StreamingContextSuite.
+ * This includes methods to access private methods and fields in ExecutorAllocationManager.
```

Comment:
I figured that would be a good place holder for future methods that may need to be included.
Can re-word accordingly.